packages/back-end/test/integrations/integration-query-test.sh

```diff
@@ -0,0 +1,4 @@
+source ~/.env
```

Comment:
Not sure the best practice for storing/sharing credentials. Right now, my credential file looks like this:

```
export MYSQL_TEST_HOST=localhost
export MYSQL_TEST_USER=XXXX
export MYSQL_TEST_DATABASE=sample
export MYSQL_TEST_PASSWORD=XXXX

export POSTGRES_TEST_HOST=localhost
export POSTGRES_TEST_USER=XXXX
export POSTGRES_TEST_DATABASE=sample

export GOOGLE_APPLICATION_CREDENTIALS="FILE"
```

Reply:
I personally like the tool [direnv](https://direnv.net/). It's an alternative to something like dotenv, which needs to be implemented at the application-level. direnv is a tool you need to have installed on your machine but it's agnostic to application frameworks or programming languages. I don't think we should be crawling into the user's home directory.

Reply:
There's also https://pypi.org/project/python-dotenv/

I agree, it should not use the home directory. We can just move the .env file to the test/integrations directory and add it to .gitignore

Reply:
Thanks, I knew this was sloppy but raised this comment precisely because I wasn't sure what a good solution was.

I think for now I would just move it into this folder and .gitignore it as @jdorn suggests.

Reply:
I think I'll also use `python-dotenv`. Thanks for linking it. That way we don't have to mess with our user's environment.

Reply:
Updated and re-requested review from @tinahollygb .